tests/headers/macros/macro_conditionals.h

```diff
@@ -50,7 +50,7 @@
 #ifdef UNDEFINED
   #define NO_DEFINE_ELIF
   int NO_DECLARE_ELIF;
-#elif defined MACRO
+#elif defined(MACRO)
```

Comment:
I added this, as
```C
#if defined(X)
```
is valid C prepocessor code, too.

But there are only tests for
```C
#if defined X
```

Now both variants are tested